### ISSUE TITLE
a bug (?) when distilling TinyBERT on regression tasks with task_distill.py

### ISSUE THREAD

**Issue by yellow-binary-tree** [huawei-noah/Pretrained-Language-Model#128]
in TinyBERT/task_distill.py line 973:
``` python
elif output_mode == "regression":
    loss_mse = MSELoss()
    cls_loss = loss_mse(student_logits.view(-1), label_ids.view(-1))
```
but TinyBERT is actually learning from the label, maybe we should use
``` python
cls_loss = loss_mse(student_logits.view(-1), teacher_logits.view(-1))
```
instead?

**Comment by XiaoqiJiao** [comment]
Thanks for pointing it out, and we will fix this bug.

Also, we've checked the original version of our codes, and confirm the results of TinyBERT in the paper are learned from the teacher's logits.

**Comment by itsucks** [comment]
It is a bug. The result of logit distillation is slightly better  on sts-b.
